go.mod

```diff
@@ -3,8 +3,9 @@ module k8s.io/kops
 go 1.20
 
 replace (
-	github.com/docker/cli => github.com/docker/cli v20.10.21+incompatible
-	github.com/docker/docker => github.com/docker/docker v20.10.21+incompatible
+	github.com/docker/cli => github.com/docker/cli v23.0.1+incompatible
+	github.com/docker/docker => github.com/docker/docker v23.0.1+incompatible
+	k8s.io/apiserver => k8s.io/apiserver v0.26.4
```

Comment:
the problem with newer library is that https://github.com/kubernetes/kops/blob/master/pkg/applylib/applyset/managedfields.go#L187 `DecodeManagedFields` function seems to be removed without deprecation? No idea is that moved to some another package or what is the case here.